back_end/src/apis/major.py

```diff
@@ -55,19 +51,17 @@ def get_major():
     return jsonify({'reason': 'success', 'result': major}), 200
```

Comment:
Because we don't to to_json in models, here we need to call to_json.
I think for this request, we are only getting one major, so we can do `'result': major.to_json() ` in the jsonify function call.